blackbox/docs/sql/system.txt

```diff
@@ -771,6 +771,16 @@ num_docs
 |                      | within a shard.                  |             |
 +----------------------+----------------------------------+-------------+
 
+min_lucene_version
+------------------
+
++------------------------+----------------------------------+-------------+
+|     Column Name        |           Description            | Return Type |
++========================+==================================+=============+
+| ``min_lucene_version`` | Shows the version of the oldest  | ``String``  |
+|                        | segment of the shard.            |             |
```

Comment:
"Shows the oldest lucene segment version used in this shard"

background: this is not always the oldest segment